src/services/stellar/trustline.service.ts

```
@@ -0,0 +1,304 @@
+      throw new StellarError('Trustline confirmation timeout', undefined, { 
+        publicKey, 
+        assetCode: asset.getCode(),
+        assetIssuer: asset.getIssuer(),
+        timeout 
+      });
+    });
+  }
+
+  validateTrustlineLimit(limit: string): boolean {
+    const numLimit = parseFloat(limit);
+    return !isNaN(numLimit) && numLimit >= 0 && numLimit <= 922337203685.4775807;
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 922337203685.4775

(lint/correctness/noPrecisionLoss)